.luacheckrc

```diff
@@ -1,5 +1,7 @@
 globals = {
     "_clean_buffer_count",
     "vim",
+    "lean_nvim_ft_options",
+    "_LEAN3_STANDARD_LIBRARY",
```

Comment:
This doesn't have to be a global I think, there's just an accidental removal of the `local` in `ftdetect/lean`, if you add it back you can remove this I suspect.